containers/base/Dockerfile.in

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:jessie
+FROM _base_image_
```

Comment:
Done for the base image.  We could do this for the top-level image as well, but that already has a template for the version numbers, so leaving it as is for now.